### PR TITLE
Big switch config

### PR DESCRIPTION
**Pull request by wanleg**
not sure how else to "invalidate" the matrix_init/scan_user functions in bigswitch.c 
seems like it might be better to move all that stuff to the keymap file anyway?

### CODE REVIEW

**Comment by drashna** [comment]
Actually, if you add `__attribute__ ((weak))` before them, then if you set them in the keymap.c file, it will be overwritten. 

That may be better here?

File: keyboards/bigswitch/keymaps/wanleg/config.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/bigswitch/keymaps/wanleg/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

**Comment by wanleg** [comment]
thanks for the tips! suggested changes made.

**Comment by drashna** [comment]
Thanks!  
(and you're welcome!)

**Pull request merged**